src/Stack/Build/Source.hs

```diff
@@ -389,7 +389,7 @@ loadLocalPackage pp = do
   componentFiles <- memoizeRefWith $
     fst <$> getPackageFilesForTargets pkg (ppCabalFP pp) nonLibComponents
 
-  checkCacheResults <- memoizeRefWith $ do
+  checkCacheResults <- memoizeRefWith . debugBracket "checkCacheResults" $ do
```

Comment:
I intend to drop this pre-merge.

Reply:
OK. I think we are close to merging - can you drop this?

Reply:
done.